serveradmin/servershell/static/js/servershell/command.js

```diff
@@ -62,7 +76,9 @@ servershell.update_attribute = function(object_id, attribute_id, new_value, mult
         let to_remove;
 
         // Don't add or remove empty values
-        new_value = new_value.filter(v => v.trim() !== '');
+        new_value = new_value.filter(function(v) {
+            return (v + '').trim() !== '';
```

Comment:
What makes it needed to add `''` to force it to a string and not e.g. using `String(v)`?

Reply:
Yes. What is the benfit of String(v) over ' ' + v ?

Reply:
Not much actually. `String(v)` looks more elegant to me. Read also this https://stackoverflow.com/a/3945236/2625488 and experimented a bit:
![image](https://user-images.githubusercontent.com/5676670/80603867-41314500-8a31-11ea-9501-d1af54b311d7.png)

Reply:
Thanks for the research. I did not knew that. 

All tough passing non primitive types is not supported by Servershell so it should not be a use case, I think `v = ''` is the better choice because if we ever come to this point we want to have the primitive representation of valueOf() and not the String representation of that object.